configs/src/deployments/json/arbitrum.json

```diff
@@ -180,5 +180,172 @@
       "blockNumber": 145462176,
       "deploymentTx": "0x6459d6c62aac34ff322dc28e18432bf8a165d2cd3f1470e1bd162689e8492e27"
     }
+  },
+  "v1.4.0": {
+    "ManagementDAOProxy": {
+      "address": "0xF3AaA3372EbBf01b923a4Cc98Cd847126b3D73cA",
+      "blockNumber": 145461969,
+      "deploymentTx": "0x775e926293f5838b4782fc24b12987cfbe472c988d421fe31d1270adca9ade3c"
+    },
+    "ManagementDAOImplementation": {
+      "address": "0xc3F1f4d3B4E24b6F019120205e12A01D733BEb55",
+      "blockNumber": 314229082,
+      "deploymentTx": "0xd4aee31350abe3fa88381d04ce5f5d6eeb12740e59869adc9d3876e518307bbc"
+    },
+    "DAOBase": {
+      "address": "0xc3F1f4d3B4E24b6F019120205e12A01D733BEb55",
+      "blockNumber": 314229082,
+      "deploymentTx": "0xd4aee31350abe3fa88381d04ce5f5d6eeb12740e59869adc9d3876e518307bbc"
+    },
+    "DAOFactory": {
+      "address": "0x49e04AB7af7A263b8ac802c1cAe22f5b4E4577Cd",
+      "blockNumber": 314229082,
+      "deploymentTx": "0xd4aee31350abe3fa88381d04ce5f5d6eeb12740e59869adc9d3876e518307bbc"
+    },
+    "DAORegistryProxy": {
+      "address": "0xB5146Fd572C669ABC353902e43F47fda4609E38A",
+      "blockNumber": 145462155,
+      "deploymentTx": "0xb954cc113df511f97407f130ec49e70b1d5e34ba6de0b1d7aeaa2de5cc8ecdfe"
+    },
+    "DAORegistryImplementation": {
+      "address": "0x7DC8d6d35070A13192C754CF96CB046615f2a3E0",
+      "blockNumber": 314229104,
+      "deploymentTx": "0x176c301166553255120102f729ef69f3ec5645b777d987188a0956cc1e2ba33e"
+    },
+    "DAOENSSubdomainRegistrarProxy": {
+      "address": "0x57bf333951967a0cC0afcD58FC7959Ca0Eae6905",
+      "blockNumber": 145462121,
+      "deploymentTx": "0xccc80fbef5cba711fc7f15dba72b90ddd46366c087e8c935f3955c9df55f12f2"
+    },
+    "DAOENSSubdomainRegistrarImplementation": {
+      "address": "0xBe6eAC1bA0b2d20AEA9e7924EFEE8E72D6193246",
+      "blockNumber": 145462095,
+      "deploymentTx": "0x979d4d97552fa985cd9aa12abec9e458023774c0456b86874fca9252507442b5"
+    },
+    "ENSRegistry": {
+      "address": "0x9277acd65B5dc0F85867Dfd40f5488Be8d47AD18",
+      "blockNumber": 145462021,
+      "deploymentTx": "0x626fe46043f38a9c4b87f6c506b2bb81e3a05a5f389559588b8a835ca20a5a0e"
+    },
+    "PlaceholderSetup": {
+      "address": "0x6E924eA5864044D8642385683fFA5AD42FB687f2",
+      "blockNumber": 145462388,
+      "deploymentTx": "0x35fd51be4013bff16d0e87c2676b06cfe7f232fd22e5673a99a49e535d4c233c"
+    },
+    "PluginRepoBase": {
+      "address": "0x2820Dc9eDED81978F62658ea2fA4eF6A1c0944F4",
+      "blockNumber": 314229094,
+      "deploymentTx": "0x9726bb9d935b55232347ef933dca350fd7302bc0fb4cde516c6050f226faddb7"
+    },
```

Comment:
Same as with Polygon, existing repo's point to `0x78744a9664d6a4ddbcd7b73c120535667e739ee3` rather than this